src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -715,6 +715,16 @@ public string Description
             }
         }
 
+        /// <summary>
+        /// Node description that can be customized by the user.
+        /// </summary>
+        [JsonIgnore]
+        public string UserDescription
```

Comment:
@nate-peters @SHKnudsen @saintentropy what do you guys think about making this internal and only exposing the viewmodel wrapper?